packages/orchestration/src/orchestration-types.ts

```diff
@@ -315,3 +320,34 @@ export interface TranslationConfigParams {
    */
   targetLanguage: string;
 }
+
+/**
+ * Filter configuration for Azure Content Safety.
+ */
+export type AzureContentSafety = AzureContentSafetyOutput;
+
+/**
+ * Representation of the Azure Content Safety filter config schema.
+ */
+export type AzureContentSafetyFilterConfig =
+  AzureContentSafetyOutputFilterConfig;
+
+/**
+ * Representation of the 'ErrorResponse' schema.
+ */
+export type ErrorResponse = OriginalErrorResponse;
+
+/**
+ * Representation of the 'ChatCompletionTool' schema.
+ */
+export type ChatCompletionTool = OriginalChatCompletionTool;
+
+/**
+ * Representation of the 'ToolChatMessage' schema.
+ */
+export type ToolChatMessage = OriginalToolChatMessage;
+
+/**
+ * Only include DataRepositories with the given type.
+ */
+export type DataRepositoryType = OriginalDataRepositoryType;
```

Comment:
[for reviewer] These are types that used in our sample-code, which requires now aliases.

Reply:
[req] Why bother here? The sample code should use the internal types as well. Its just sample. We need to show the correct way to import types (which is from internal for generated types), else we're just confusing people.